core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.html

```diff
@@ -1,35 +1,32 @@
 <div class="oppia-opportunities-list">
   <div [hidden]="loadingOpportunityData" *ngFor="let opportunity of visibleOpportunities">
-    <oppia-opportunities-list-item [opportunity]="opportunity"
-                                   (clickActionButton)="clickActionButton.emit($event)"
-                                   [progressBarRequired]="progressBarRequired"
-                                   [labelRequired]="labelRequired"
-                                   [opportunityHeadingTruncationLength]="opportunityHeadingTruncationLength"
-                                   [opportunityType]="opportunityType"
-                                   [showOpportunityButton]="showOpportunityButton">
+    <oppia-opportunities-list-item [opportunity]="opportunity" (clickActionButton)="clickActionButton.emit($event)"
```

Comment:
@Ash-2k3 I think your PR has a bunch of superfluous changes. Please leave the file as-is, as much as possible, and only change the parts which actually do need changing. This makes it easier for reviewers to see what changed.

Could you please revise this accordingly, and then make a follow-up review request? Thanks.

Reply:
Thanks a lot for marking this...I have fixed this and I will keep this mind for future contributions..
Can you review it again please ?
Thanks.